stdlib/types.pyi

```diff
@@ -350,6 +369,9 @@ DynamicClassAttribute = property
 
 def coroutine(func: Callable[..., Any]) -> CoroutineType: ...
 
+if sys.version_info >= (3, 8):
+    CellType = _Cell
+
```

Comment:
`CellType` was made public in [python 3.8](https://docs.python.org/3/library/types.html#types.CellType).